packages/button/src/Button.tsx

```diff
@@ -46,6 +48,7 @@ function Button(props: ButtonProps, ref?: ForwardedRef<HTMLButtonElement>) {
       disabled={disabled || loading}
       type='button'
       ref={ref}
+      href={href}
```

Comment:
It's not the best way from types standpoint, because <button> props and <a> props are different:

<img width="490" alt="image" src="https://user-images.githubusercontent.com/103929444/180457245-7dca08a8-d0ff-42fa-a5de-39c48e134c5e.png">

<img width="484" alt="image" src="https://user-images.githubusercontent.com/103929444/180457328-95248b3e-e7ff-4713-8747-906a31f43c26.png">

So basically we need that button should accept button props, and link should accept link props

Reply:
What do you suggest in this situation?

Reply:
I personally like this approach, it is nice and minimal - by providing `href` you expect it to still be same component visually while making it a link. however, you are poiting different thing right - e.g. `target` attribute should be also supported here.